atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/ClientOptions.java

```diff
@@ -36,8 +36,8 @@
     // TODO (jkong): Re-enable client QoS after response body leaks are handled correctly.
     // Throws after expected outages of 1/2 * 0.01 * (2^11 - 1) = 10.24 s
     public static final ClientOptions DEFAULT_RETRYING = ImmutableClientOptions.builder()
-            .connectTimeout(Duration.ofSeconds(10))
-            .readTimeout(Duration.ofSeconds(65))
+            .connectTimeout(Duration.ofSeconds(1))
+            .readTimeout(Duration.ofSeconds(10))
```

Comment:
Same here, we should probably lower even more, as this prolongs any outage: timelock is generally fast, so even seconds here are probably too high.

Reply:
So the problem here is that lock V1 API blocks until the lock is granted, so it is legitimate.

Reply:
they don't use conjure or atlasdb-feign, they have their own remoting layer

Reply:
@felixdesouza Lock V1 is actually used by anyone who does `runTaskWithLocks(...)` - there are some users in internal metal product that explicitly take out lock-v1 locks too.

Reply:
I think this is might be the actual thing that prolongs outages though. Since Timelock runs on http2, the connection would've been already established. I think we need a bit of testing of the client side to see what this actually does.